py/picca/test/test_1_deltas.py

```diff
@@ -0,0 +1,435 @@
+import unittest
+import numpy as np
+import fitsio
+import healpy
+import os
+import tempfile
+import shutil
+import glob
+from pkg_resources import resource_filename
+import sys
+
+from picca.utils import userprint
+
+from .test_helpers import update_system_status_values, compare_fits, compare_h5py, send_requirements, load_requirements
```

Comment:
why not add these test_helpers into an AbstractTest class and have the other classes inherit from it?

Reply:
Great idea! Didn't think about it when doing that change, guess I need to work on my OOP intuition ;-)